backend/src/feeds/feed.entity.ts

```diff
@@ -22,6 +23,7 @@ export class Feed {
     @Column()
     description: string;
 
+    @Index()
```

Comment:
Why is source field used as an index here?. If you wanted to make it unique then you have to give `@Index({ unique: true })`and since you have already specified `@Unique(['source', 'pub_date'])` at entity level this is not needed or redundant.

Reply:
This is because we may query feeds using source type a lot. Like query all feeds from Facebook. Having an index will make the process faster